Gemfile

```diff
@@ -37,14 +39,29 @@ gem 'bootsnap', '>= 1.1.0', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'pry'
+  gem 'travis'
+  gem 'rspec-rails', '~> 4.0.1'
+  gem 'factory_bot_rails'
+  gem 'faker'
+  gem 'shoulda-matchers'
 end
 
 group :development do
   # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
   gem 'web-console', '>= 3.3.0'
   gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'rubocop-rails'
 end
 
+group :test do
+  gem 'capybara'
+  gem 'launchy'
+  gem 'simplecov'
+  gem 'vcr'
```

Comment:
I don't think we are using vcr due to our endpoint limit of 1000. Sticking with webmock